integration/bun/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "replicate-app-bun",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Bun integration tests",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "tsc && node --test ./dist/index.test.js"
+  },
+  "dependencies": {
+    "@types/node": "^20.11.0",
+    "replicate": "file:../../",
+    "typescript": "^5.3.3"
+  },
+  "devDependencies": {
+    "@types/bun": "latest"
+  },
+  "peerDependencies": {
+    "typescript": "^5.0.0"
+  }
+}
```

Comment:
I think we can rely on `integration/typescript` to run `tsc` if Bun doesn't do any typechecking itself then let's skip it.

Reply:
`bun init` adds `typescript` as a `peerDependency`. But looking around, I can't find a good explanation why. So I agree, we can remove this.